src/global/helpers/arrow-navigation.ts

```diff
@@ -47,8 +53,8 @@ export function getNextElementIndex(event: KeyboardEvent, current: number, size:
   }
 
   if (event.key === prevKey || event.key === 'ArrowUp') {
```

Comment:
How can you come with "-1" for param `current` into this function? Should this not maybe solved at caller side?

Reply:
In the current implementation of the autocomplete '-1' is considered as 'no active element'.
But in general, I thought this method as a global utility that might face different needs. So the more robust it is, the better.

Reply:
Check if is possible a refactoring of the others usage